graph/include/metadata/MetaData.h

```diff
@@ -36,8 +36,8 @@ class MetaDataFactory {
   template <class... T>
   static CRTPBase* create(const std::string& s, const nlohmann::json& j) {
     if (data().find(s) == data().end()) {
-      MCGLogger::instance().getErrConsole()->template warn(
-          "Could not create: {}, the Metadata is unknown in you application", s);
+      MCGLogger::instance().getErrConsole()->warn("Could not create: {}, the Metadata is unknown in you application",
+                                                  s);
```

Comment:
I would prefer to have the error in one single line, instead of having only the formatting parameter in a new line.
```suggestion
      MCGLogger::instance().getErrConsole()->warn(
          "Could not create: {}, the Metadata is unknown in you application",  s);
```

Reply:
I'm inclined to agree. Unfortunately clang-format does not and automatically changes it back to the original version :smile: